Makefile

```diff
@@ -143,13 +143,22 @@ src/ocamlbuildlib.cmxa: src/ocamlbuild_pack.cmx $(EXTRA_CMX)
 
 # The packs
 
+# Build artifacts are first placed into tmp/ to avoid a race condition
+# described in https://caml.inria.fr/mantis/view.php?id=4991.
```

Comment:
@gasche It doesn't work. The ocamlopt command touches the cmi file, leading to a spurious rebuild afterwards.